test/Test/Iris/Cli.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE ScopedTypeVariables #-}
+module Test.Iris.Cli (cliSpec) where
+
+import Test.Hspec (Spec, describe, it)
+
+import Iris.Cli.ParserInfo (cmdParserInfo)
+import Iris.Settings (defaultCliEnvSettings, cliEnvSettingsVersionSettings)
+import Iris.Cli.Cmd (Cmd (..))
+import Iris.Cli.Version (defaultVersionSettings)
+import qualified Options.Applicative as Opt
+import Data.List (isInfixOf)
+import qualified Paths_iris as Autogen
+
+
+
+type HelpOptions = [String]
+
+cliSpec :: Spec
+cliSpec = describe "Cli Options" $ do
+    let parserPrefs :: Opt.ParserPrefs = Opt.defaultPrefs 
+    it "help without version environment should output the options help and no-input and should not output the options version and numeric-version" $ do
+        let parserInfo :: Opt.ParserInfo (Cmd ()) = cmdParserInfo defaultCliEnvSettings
+        let result :: Opt.ParserResult (Cmd ()) = Opt.execParserPure parserPrefs parserInfo ["--help"]
```

Comment:
I think you don't need these type annotations 🤔 

Haskell has type global type inference so it should be able to infer most (if not all) of the types.

Generally, adding types may help the compiler when types are ambiguous. But I think they are pretty unambiguous in this case.

Reply:
Hehe they were mainly for helping myself. I will remove them.